packagelist/debian_12_headless.txt

```diff
@@ -57,6 +57,7 @@ can-utils
 gpiod
 i2c-tools
 libiio-utils
+libinput10
```

Comment:
Instead of doing this, can you make a task that is of type "packagelist" that echos "libinput10"?  Eg:
https://github.com/embeddedTS/distro-seed/blob/main/tasks/image_configuration/update-command-not-found/manifest.yaml#L6
https://github.com/embeddedTS/distro-seed/blob/main/tasks/image_configuration/update-command-not-found/packages.sh